cmd/soroban-rpc/internal/db/migration.go

```diff
@@ -7,20 +7,23 @@ import (
 
 	"github.com/stellar/go/support/log"
 	"github.com/stellar/go/xdr"
+)
 
-	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/config"
+const (
+	transactionsMigrationName = "TransactionsTable"
+	eventsMigrationName       = "EventsTable"
 )
 
 type LedgerSeqRange struct {
-	firstLedgerSeq uint32
-	lastLedgerSeq  uint32
+	FirstLedgerSeq uint32
+	LastLedgerSeq  uint32
```

Comment:
suggestion/nit: this should just be `First`, `Last` (vscode lets you "Rename Symbol" these really easily) since `LedgerSeq` is implied by the structure name